packages/web-components/src/components/tabs-extended/tabs-extended.ts

```diff
@@ -261,6 +304,15 @@ class DDSTabsExtended extends MediaQueryMixin(
         }
       });
     }
+
+    if (changedProperties.has('_isLargeOrGreater')) {
+      const { _isLargeOrGreater, _tabItems } = this;
```

Comment:
These variables have already been declared at the beginning of this method, so I think you can remove this line.

Reply:
I have removed that line! Can you please review it now @jkaeser